README.md

```diff
@@ -245,7 +243,7 @@ kind: MachineSet
 metadata:
   labels:
     machine.openshift.io/cluster-api-cluster: <infrastructureID> 
-  name: <infrastructureID>-worker-<zone>
+  name: <infrastructureID>-windows-worker-<zone>
```

Comment:
Sorry to not catch it earlier, do we not need to make this change in Labels ? 
https://github.com/openshift/windows-machine-config-operator/pull/117/files#diff-04c6e90faac2675aa89e2176d2eec7d8L255

https://github.com/openshift/windows-machine-config-operator/pull/117/files#diff-04c6e90faac2675aa89e2176d2eec7d8R260